components/event-handler-notification/org.wso2.carbon.identity.event.handler.notification/src/main/java/org/wso2/carbon/identity/event/handler/notification/NotificationConstants.java

```diff
@@ -18,13 +18,23 @@
 
 package org.wso2.carbon.identity.event.handler.notification;
 
+import org.wso2.carbon.identity.core.util.IdentityUtil;
+
 public class NotificationConstants {
 
     private NotificationConstants() {
     }
 
+    private static String getClaimUriLocale() {
+        if ("true".equals(IdentityUtil.getProperty("isLegacyLocalityClaimUri"))) {
```

Comment:
better to use Boolean.parseBoolean(IdentityUtil.getProperty("isLegacyLocalityClaimUri"))

Reply:
also first check whether IdentityUtil.getProperty("isLegacyLocalityClaimUri") is null, if so, assign the default value.

Reply:
commit c6f1d06820885092027e0f28adc4ac8b890cce3e